rally_ovs/plugins/ovs/scenarios/ovn_sandbox.py

```diff
@@ -143,8 +144,84 @@ def delete_sandbox(self, sandbox_delete_args=None):
         self._delete_sandbox(sandboxes, graceful)
 
 
+    def _get_sandbox(self, farm="", tag=""):
+        if "sandboxes" in self.context:
+            ret = self.context["sandboxes"]
+        else:
+            ret = utils.get_sandboxes(self.task["deployment_uuid"], farm, tag)
+
+        return ret
+
+
+    @scenario.configure(context={})
+    def start_sandbox(self, sandbox_start_args=None):
+        """Start sandboxes specified by 'sandbox_stop_args' or 'sandbox' context.
+        If they are already running, do nothing.
+
+        :param sandbox_start_args: dict, contains below values:
+
+            ===========    ========
+            key            desc
+            ===========    ========
+            farm           str, the name of farm node
+            tag            str, a tag used to identify a set of sandboxes
+            ===========    ========
+
+        If both 'farm' and 'tag' are not set in 'sandbox_start_args',
+        'sandbox' context will be used to find sandboxes.
```

Comment:
Does this comment contradict to the implementation in _get_sandbox()? In _get_sandbox(), the if condition checks self.context first.